tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,61 @@
 fig.gca().view_init(azim=70, elev=15)
 ten_twenty_montage.plot(kind='topomap', show_names=False)
 
+###############################################################################
+# .. _control-chan-projection:
+#
+# Controlling channel projection (MNE vs EEGLAB)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Channel positions in 2d space are obtained by projecting their actual 3d
+# positions using a sphere as a reference. Because ``'standard_1020'`` montage
```

Comment:
@larsoner I changed this to "using a sphere as a reference" - this may be less clear, but is closer to what we do. After all it was funny to mention that channels are projected to a sphere in a PR titled "do not project to sphere" 😀